src/components/Calculator.js

```diff
@@ -1,48 +1,39 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
 const operators = ['÷', 'x', '-', '+', '='];
 const digits = ['AC', '+/-', '%', '7', '8', '9', '4', '5', '6', '1', '2', '3', '0', '.'];
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      next: '',
-      total: '',
-      operation: '',
-    };
-  }
+const Calculator = () => {
+  const [obj, calcObj] = useState({
+    next: '',
+    total: '',
+    operation: '',
+  });
```

Comment:
- [ ] You did well refactoring your app but for now, it is not performing the calculation, please fix that